src/Widgets/DisplayWidget.vala

```diff
@@ -253,42 +253,42 @@ public class Display.DisplayWidget : Gtk.EventBox {
 
             switch (transform) {
                 case DisplayTransform.NORMAL:
-                    virtual_monitor.get_current_mode_size (out real_width, out real_height);
+                    virtual_monitor.get_current_mode_size (out _real_width, out _real_height);
```

Comment:
Can this line not be moved to before the switch statement to avoid repetition?

Reply:
1. Notice that the order of the arguments `real_width` and `real_height` are swapped every other case. So I have no idea how one could avoid the repetition here.

2. Ok, I understand. There already exists a public getter function `get_geometry` which I should have used from the beginning.

Reply:
Even better - I missed that too.